src/sentry/static/sentry/app/views/settings/components/dataScrubbing/modals/localStorage.tsx

```diff
@@ -0,0 +1,44 @@
+import * as Sentry from '@sentry/browser';
+
+import localStorage from 'app/utils/localStorage';
+
+import {SourceSuggestion} from '../types';
+
+const ADVANCED_DATA_SCRUBBING_LOCALSTORAGE_KEY = 'advanced-data-scrubbing';
+
+type StorageValue = {
+  eventId: string;
+  sourceSuggestions: Array<SourceSuggestion>;
+};
+
+// TODO(Priscila): add the method below in app/utils
+const fetchFromStorage = (): StorageValue | undefined => {
```

Comment:
I was not aware we're storing stuff in local storage. Why do we do that?

Reply:
because now I am using the GlobalModal and I cannot save the eventId outside this modal while I am using it.. I tried a couple of things that I can explain later, but this was the best solution I came with + the user now can switch routes and when he/she/they will be back, the event id will be still available